tox.ini

```diff
@@ -20,6 +20,139 @@ setenv =
   PEP517_OUT_DIR = {env:PEP517_OUT_DIR:{toxinidir}{/}dist}
 
 
+[testenv:build-docs]
+allowlist_externals =
+  git
+basepython = python3
+commands_pre =
+  # Paramiko:
+  {envpython} -m pip install -r{toxinidir}/requirements-tunnel.txt
```

Comment:
This is not in the deps because the docs requirements use a constraints file with hashes and when pip sees at least one hash, it requires all of the packages to be listed with hashes. The one on this line doesn't have hashes and will fail if listed in the same command. It's a hack but it is what it is. If the env deps infra is properly converted to pip-tools managed constraints files, it would be possible to move this to a better place.